### PR TITLE
test(rtl): refactor commands-bisect component test

### PR DESCRIPTION
**Pull request by erickzhao**
Replaces the old `commands-bisect-spec.tsx` Enzyme test with new shiny RTL ones.

Along the way, there are a couple non-test changes to note:
* New dependency: `@testing-library/user-event` is apparently the way to go to simulate user events over the built-in `fireEvent` helper method.
* I added new `aria-label` properties to the 👍 and 👎 buttons. This makes the buttons more accessible!

### CODE REVIEW

File: tests/mocks/state.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export class StateMock {
   public electronMirror = ELECTRON_MIRROR;
   public isBisectCommandShowing = false;
 
-  public Bisector = new BisectorMock();
+  public Bisector: BisectorMock | undefined = new BisectorMock();
```
Comment (BlackHole1) [inline tests/mocks/state.ts:62]: Under what circumstances will undefined be returned here?

File: tests/renderer/components/__snapshots__/commands-bisect-spec.tsx.snap [hunk 2]
```diff
@@ -1,51 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (BlackHole1) [inline tests/renderer/components/__snapshots__/commands-bisect-spec.tsx.snap:1]: Looks like we won't be needing snapshots anymore, right?

**Review by BlackHole1 — COMMENTED** [review]
LGTM. A very great change!

It seems that there will be no more unit test failures caused by new contributors changing the UI layout and forgetting to execute `yarn run test -- -u`.

File: tests/mocks/state.ts (hunk 1 above)
Comment (erickzhao) [reply tests/mocks/state.ts:62]: `Bisector` is undefined if there is no active bisect running. This is tested under the `'renders bisect dialog button if no bisect instance'` test!

File: tests/renderer/components/__snapshots__/commands-bisect-spec.tsx.snap (hunk 2 above)
Comment (erickzhao) [reply tests/renderer/components/__snapshots__/commands-bisect-spec.tsx.snap:1]: We could always use inline Jest snapshots to explicitly test UI bits, but I don't see us using large snapshot files.

Comment (BlackHole1) [reply tests/renderer/components/__snapshots__/commands-bisect-spec.tsx.snap:1]: Okey

**Review by BlackHole1 — APPROVED** [review]

**Pull request merged**